### PR TITLE
Sanity Yoga Class Schema

### PR DESCRIPTION
**Pull request by anthopark**
Defined a schema for Yoga class
* Name
* Duration in minutes
* Level
* Monthly Price

### CODE REVIEW

**Comment by anthopark** [comment]
[Define "Yoga Class" Sanity Schema](https://trello.com/c/P7VhhWuh/7-define-yoga-class-sanity-schema)

File: sanity-studio/schemas/yoga-class.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+export default {
+  title: 'Yoga Class',
+  name: 'yogaClass',
+  type: 'document',
+  fields: [
+    {
+      title: 'Name',
```
Comment (emilyPhee) [inline sanity-studio/schemas/yoga-class.js:7]: the title was human-readable property?

Comment (emilyPhee) [inline sanity-studio/schemas/yoga-class.js:7]: the title was human-readable property?

Comment (anthopark) [reply sanity-studio/schemas/yoga-class.js:7]: yes

**Pull request merged**